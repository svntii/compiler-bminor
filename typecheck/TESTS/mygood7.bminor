//comment i dont know how to fix the dangling else

a:function void() = {
    x:integer = 10;
    if(x>1)
    {
        print "1";
    }

}


//last comment