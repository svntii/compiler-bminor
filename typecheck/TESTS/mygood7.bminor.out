//	CSE 40243 B-Minor Parser
//	Enter a valid  B-Minor Code 

//	Scan Successful
//	Parse Successful
a: function void()  = 
{
    if (x > 1)
        
        {
            print "1";
            
        }
        
}

